server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/BaseRuntimeFlowableDelegate.java

```diff
@@ -75,7 +75,7 @@ public abstract class BaseRuntimeFlowableDelegate<T> extends BaseFlowableDelegat
     @Getter
     private TaskType taskType;
     @Getter
-    private Long flowInstanceId;
+    protected Long flowInstanceId;
```

Comment:
Will this field be modified?

Reply:
flowInstanceId will be set by inherit class OnlineSchemaChangeFlowTask.tryComplete() function and this class is created when use in Scheduler